model/accounts_mgmt/v1/cluster_metrics_nodes_type.model

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+class ClusterMetricsNodes {
```

Comment:
The `class` types are intended for things that have their own identity, and a resource in the API that manages them. For example, a cluster, a subscription, an account, etc. But I think this is not the case, so it should be a `struct` instead.

Reply:
fixed